core/src/test/java/apoc/meta/MetaTest.java

```diff
@@ -1539,4 +1540,36 @@ public void testMetaNodesCount() {
         
     }
 
+    @Test
+    public void testRelTypePropertiesMovies() throws Exception {
+        final String query = IOUtils.toString(new InputStreamReader(Thread.currentThread().getContextClassLoader().getResourceAsStream("movies.cypher")));
```

Comment:
You could use `getClass().getClassLoader()` instead of `Thread.currentThread().getContextClassLoader()`, I believe. I slightly prefer  it since it is simpler and used in other places, although it is not a big deal.

Reply:
ok